src/NodaTime/Duration.cs

```diff
@@ -1123,6 +1124,13 @@ public static Duration FromTimeSpan(TimeSpan timeSpan)
         public TimeSpan ToTimeSpan() => new TimeSpan(BclCompatibleTicks);
 
         #region XML serialization
+        /// <summary>
+        /// Adds the XML schema type describing the structure of the <see cref="Duration"/> XML serialization to the given <paramref name="xmlSchemaSet"/>.
+        /// </summary>
+        /// <param name="xmlSchemaSet">The XML schema set provided by <see cref="XmlSchemaExporter"/>.</param>
+        /// <returns>The qualified name of the schema type that was added to the <paramref name="xmlSchemaSet"/>.</returns>
+        public static XmlQualifiedName GetSchema(XmlSchemaSet xmlSchemaSet) => XmlSchemaDefinition.AddDurationSchemaType(xmlSchemaSet);
```

Comment:
Is it still correct for IXmlSerializable.GetSchema to return null below?

Reply:
Yes, here's an excerpt from the remarks of the [IXmlSerializable.GetSchema](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.ixmlserializable.getschema) documentation (emphasis mine):
> When serializing or deserializing an object, the [XmlSerializer](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.xmlserializer) class does not perform XML validation. For this reason, it is often safe to omit schema information by providing a trivial implementation of this method, for example by returning `null` (`Nothing` in Visual Basic).
> 
> If providing schema information is necessary, you should use the [XmlSchemaProviderAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.xmlschemaproviderattribute) attribute. **The `GetSchema` method should still return `null` (or `Nothing`)**.